FrameworkSystem/scripts/dirac-proxy-info.py

```diff
@@ -1,16 +1,21 @@
 #!/usr/bin/env python
 ########################################################################
-# $HeadURL$
 # File :    dirac-proxy-init.py
 # Author :  Adrian Casajus
 ########################################################################
-__RCSID__ = "$Id$"
-
 import sys
-import os.path
+
 import DIRAC
 from DIRAC.Core.Base import Script
 from DIRAC.Core.Utilities.NTP import getClockDeviation
+from DIRAC import gLogger
+from DIRAC.Core.Security.ProxyInfo import getProxyInfo, getProxyStepsInfo, formatProxyInfoAsString, formatProxyStepsInfoAsString
+from DIRAC.Core.Security import VOMS
+from DIRAC.FrameworkSystem.Client.ProxyManagerClient import gProxyManager
+from DIRAC.ConfigurationSystem.Client.Helpers import Registry
```

Comment:
According to the command writing rules, import of DIRAC modules must be done after the parseCommandLine() call